assets/src/edit-story/components/helpCenter/useHelpCenter.js

```diff
@@ -61,6 +62,11 @@ const initial = {
     hasBottomNavigation: false,
     isPrevDisabled: true,
     isNextDisabled: false,
+    read: [],
+    fetchError: false,
+    readError: false,
+    loading: false,
+    updating: false,
```

Comment:
I think we can name these a little closer to the purpose they have. right now we're interacting with the endpoint in two separate ways:
1) populating read tips
2) persisting read tips

based of off reading these, it doesn't really tell me what state is related to which action